odra-casper/livenet-env/src/livenet_host_env.rs

```diff
@@ -0,0 +1,174 @@
+use std::sync::{Arc, RwLock};
+use std::thread::sleep;
+
+use odra_casper_client::casper_client::CasperClient;
+use odra_core::{
+    Address, Bytes, CallDef, ContractEnv, EntryPointsCaller, HostContext, OdraError, PublicKey,
+    RuntimeArgs, U512
+};
+use odra_core::callstack::{Callstack, CallstackElement, Entrypoint};
+use odra_core::contract_container::ContractContainer;
+use odra_core::contract_register::ContractRegister;
+use odra_core::event::EventError;
+use odra_core::event::EventError::CouldntExtractEventData;
+use odra_core::prelude::*;
+
+use crate::livenet_contract_env::LivenetContractEnv;
+
+pub struct LivenetEnv {
```

Comment:
```suggestion
pub struct LivenetHostEnv {
```

Reply:
More like LivenetHost - so it will be similar to other hosts.